pep-0007.txt

```diff
@@ -124,11 +124,13 @@ Code lay-out
                    type->tp_name);
 
 * When you break a long expression at a binary operator, the
-  operator goes at the end of the previous line, e.g.::
+  operator goes at the end of the previous line, and braces should be
```

Comment:
Actually I think the brace should be moved only if the indentation of continuation lines matches the indentation of the following code. In the following case the brace should be kept at the end of the line: 
```
        if (PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
                             "invalid escape sequence '\\%c'",
                             *first_invalid_escape) < 0) {
            Py_DECREF(result);
            return NULL;
        }
```